### ISSUE TITLE
[FEAT] Vim support

### ISSUE THREAD

**Issue by Palanikannan1437** [gitpod-samples/Gitpod-Raycast-Extension#52]
How about some VIM support building upon @axonasif's https://github.com/axonasif/dotsh configuration steps 🚀

**Comment by axonasif** [comment]
VIM sounds really interesting! I'm wondering if you could share a bit more about how this feature?

**Comment by henit-chobisa** [comment]
> VIM sounds really interesting! I'm wondering if you could share a bit more about how this feature?

Current plan of thought is to aim for two things, 

## SSHing to the workspace
As you know with the latest beta, we are able to start the workspaces, and with the workspace url we can open it anywhere, we tried sshing method today with ssh://{workspace_url}, which was a success and opens nvim with iterm and kitty 😄 (after we setup dotsh files building upon your configuration)

## Setting up user's dotfiles in Gitpod(nvim for now) by using user's local config

Our extension has access to two significant things for this task, 
1. User's GitHub 
2. User's System 

So basically the task being done with your dotsh prompt, we can do it with the help of a GUI interface for folks to easily get started with (of Raycast for now) within the extension

Hence we can just automate it with the default nvim config of the user's machine  or if the user just wants to get started we will use options like lvim/some popular configs building upon your dotsh system and push it and automatically add this to their Gitpod preferences.

**Comment by Palanikannan1437** [comment]
And here's a quick demo of how seamless it'll be to use Vim/any terminal editor with Gitpod 🧡(please refer to the above comment for implementation details)

Start Workspace --> Launch Gitpod in your editor (acc to dotfiles) directly in your terminal without the whole browser loop

https://user-images.githubusercontent.com/73993394/232240942-a1479740-df09-4927-973c-e8794b64095d.mov

CC: @loujaybee @pawlean @axonasif
